hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestDataGenerator.java

```diff
@@ -140,35 +142,64 @@
   public static final TypeDescription ORC_TRIP_SCHEMA = AvroOrcUtils.createOrcSchema(new Schema.Parser().parse(TRIP_SCHEMA));
   public static final Schema FLATTENED_AVRO_SCHEMA = new Schema.Parser().parse(TRIP_FLATTENED_SCHEMA);
 
-  private static final Random RAND = new Random(46474747);
+  private final Random r;
```

Comment:
I actually tried to search the variable and see if anything is missed.  `r` gives me hard time :) Besides, I usually use `rand` and try to avoid single character variable.  More like a style thing, not strong opinion.

Reply:
Fair enough